lib/connection/statement.js

```diff
@@ -29,23 +30,70 @@ var statementTypes =
     FILE_PRE_EXEC: 'FILE_PRE_EXEC'
```

Comment:
We don't need a `FILE_POST_EXEC` flag here for when we're done with PUT command executing?

Reply:
I'll create a FILE_POST_EXEC flag. 

The ROW_POST_EXEC flag is used to check if Parameters.update() should be called or not, I'll add FILE_POST_EXEC to that check as well.